UnitedIncome/components
# [APP-3257] ItemizationBox Component

### Description
> Please provide a brief description of your changes below.

Implements the `<ItemizationBox />` component. This component itemizes a list of values/labels and creates a total at the top. You can collapse the component to reveal each itemization, and also include an optional button and tooltip. Additionally if the value goes into the negatives it will be reflected with `$pink`.

<img width="750" alt="Screen Shot 2019-11-22 at 11 20 08 AM" src="https://user-images.githubusercontent.com/10888441/69442719-0f49e300-0d1b-11ea-8121-629f999a8c2a.png">
<img width="758" alt="Screen Shot 2019-11-22 at 11 20 22 AM" src="https://user-images.githubusercontent.com/10888441/69442720-0f49e300-0d1b-11ea-8746-586953ec406c.png">
<img width="751" alt="Screen Shot 2019-11-22 at 11 20 30 AM" src="https://user-images.githubusercontent.com/10888441/69442721-0f49e300-0d1b-11ea-8298-ea64b36dff7f.png">


### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Load up the component library and look at the `ItemizationBox` component.
* Ensure the values line up, and that it matches the comp.

### Comments
> Any additional comments that will aid with the review process.

I will verify with Monique to make sure everything looks good with her before merging.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3257)
